.gitignore

```diff
@@ -52,6 +52,7 @@ android/app/gradle/
 android/app/gradlew
 android/app/gradlew.bat
 android/*.jks
+android/rn-libs/**/*.iml
```

Comment:
We already have files that match this, don't we? What do we expect to happen to those?

Reply:
Actually the existing needs to stay, but we need to stop tracking further changes on those, but I think we'll still need to remove them for complete fix, but let's stick do it in other PR's.

Reply:
OK?

Reply:
Sounds good :+1: